### DISCUSSION TITLE
API - numero documento

### DISCUSSION THREAD

**Discussion by mtamrc87** [fattureincloud/api#447] [Q&A] [answered]
E' possibile visualizzare tramite API il numero del documento? sia delle fatture passive sia di quelle attive.

**Comment by valmoz** [comment, marked as answer]
Buongiorno @mtamrc87 
tramite le chiamate di tipo List e Get è possibile ottenere il campo indicato.

Ad esempio, per gli issued documents:
- [List Issued Documents](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents)
- [Get Issued Document](https://developers.fattureincloud.it/api-reference#get-/c/-company_id-/issued_documents/-document_id-)

Dalla API Reference può recuperare lo schema restituito dalle diverse chiamate.

A disposizione per ulteriori domande.
Mauro

**Reply by valmoz** [reply]
Buongiorno @mtamrc87 
ho risposto alla sua domanda?

Grazie
Mauro
